src/posts/lucky_1.0.0-rc1.cr

```diff
@@ -0,0 +1,239 @@
+    you'll now have to include it as an additional dependency. Be sure to add it
+    to your `shard.yml` file, and require it in your `src/shards.cr`.
+
+    ### Support for Array params in operations
+
+    Support for Arrays in Lucky and Avram have existed for a few versions now.
+    This includes support for arrays in params. However, one bit that has been
+    missing was the ability to pass those array params through to an operation.
+
+    For example:
+
+    ```crystal
+    # before this release
+    # post:tags[]=lucky&post:tags[]=crystal
+    tags = params.nested_arrays("post")["tags"]
+    SavePost.create(params, tags: tags) do |o, p|
+      # ...
+    end
+
+    # after this release
+    SavePost.create(params) do |o, p|
+      # ...
+    end
+    ```
+
+    This ended up being quite a big breaking change. The original Params implementation
+    assumed the type was `Hash(String, String)`, but this meant you couldn't have an
+    array value. Now the type assumes `Hash(String, Array(String))` which is actually
+    the same type as [`URI::Params`](https://crystal-lang.org/api/1.5.1/URI/Params.html#new%28raw_params%3AHash%28String%2CArray%28String%29%29%29-class-method).
+
+    ### Changes to the javascript Ecosystem
+
+    As we all know, the javascript ecosystem changes in leaps and bounds daily.
+    This makes it very difficult to keep up.
+
+    In 2017, when Lucky began, the Rails core team had a neat library called
+    [Turbolinks](https://github.com/turbolinks/turbolinks-classic).
+    This library helped to make pages feel even faster, and who doesn't love speed?
+    Lucky included this library by default, but it has since been deprecated in favor
+    of [Turbo](https://turbo.hotwired.dev/). Although Lucky doesn't include Turbo currently,
+    we would still like to encourage users to give it a shot, and let us know if this
+    is something you'd like to see included by default.
+
+    Along those same lines, Lucky also came out-of-the-box with [Laravel-Mix](https://laravel-mix.com/)
+    which is "An elegant wrapper around Webpack". This made bringing javascript to Lucky
+    seemless, and easy. However, the Laravel framework has [moved to Vite](https://laravel-news.com/vite-is-the-default-frontend-asset-bundler-for-laravel-applications)
+    as their default front-end. Laravel-Mix may or may not stick around, but with the
+    uncertianty, the 1.0.0-rc1 release opens up the possibility to use [Vite](https://vitejs.dev/)
+    while keeping with the type-safe traditions Lucky brings.
+
+    So why even include a javascript build system with Lucky?
+
+    One of Lucky's goals is to be type-safe and help catch bugs in development. This
+    is done many ways, but one of the most important ways is when it comes to loading
+    your assets in production. By utilizing a generated manifest file, Crystal can
+    generate an internal Hash at compile-time that can be referenced in the app.
```

Comment:
I _believe_ it is a NamedTuple since Hash does not have compile-time safety for keys

```suggestion
    generate an internal `NamedTuple` at compile-time that can be referenced in the app.
```